packages/kbn-esql-validation-autocomplete/src/definitions/functions.ts

```diff
@@ -668,8 +668,40 @@ const dateExtractDefinition: FunctionDefinition = {
       params: [
         {
           name: 'datePart',
-          type: 'chrono_literal',
+          type: 'string',
           optional: false,
+          literalOptions: [
```

Comment:
Why are we repeating them here and not adding them to a common place that can be reused?

Reply:
Do you mean define an array and reuse it between the generator script and the file it generates (this file)?

Reply:
Ja!!